xtask/src/main.rs

```diff
@@ -160,7 +159,8 @@ fn show() -> Result<(), DynError> {
         .allowlist_function("Rf_isEnvironment")
         .allowlist_function("Rf_eval")
         .allowlist_var("R_GlobalEnv")
-        .allowlist_function("Rf_findVarInFrame3")
+        .allowlist_function("Rf_findVarInFrame")
```

Comment:
As quoted on https://github.com/yutannihilation/savvy/issues/269#issuecomment-2184497268, `Rf_findVarInFrame` might be removed in future as well. But anyway there's no alternative yet.